dotCMS/src/main/java/com/dotmarketing/portlets/templates/business/TemplateAPIImpl.java

```diff
@@ -309,6 +309,9 @@ public Template saveTemplate(Template template, Host destination, User user, boo
 
 	public List<ContainerUUID> getContainersUUID(TemplateLayout layout) {
 		final List<ContainerUUID> containerUUIDS = new ArrayList<>();
+		if (layout == null) {
+			return containerUUIDS;
```

Comment:
I would change it to Collections.emptyList();
and give memory to containerUUIDS later